app/src/androidTest/java/com/github/onlynotesswent/ui/overview/OverviewTest.kt

```diff
@@ -248,6 +249,37 @@ class OverviewTest {
     verify(navigationActions).navigateTo(screen = Screen.EDIT_NOTE)
   }
 
+  @Test
+  fun openFileSystem() {
+    composeTestRule.onNodeWithTag("FileSystemPopup").assertIsNotDisplayed()
+    composeTestRule.onNodeWithTag("showFileSystemButton").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("showFileSystemButton").performClick()
+    composeTestRule.onNodeWithTag("FileSystemPopup").assertIsDisplayed()
+  }
+
+  @Test
+  fun navigateFileSystem() {
+    val userRootFoldersFlow =
+        listOf(
+            Folder(id = "8", name = "Root Folder 1", userId = "1", parentFolderId = "1"),
+            Folder(id = "9", name = "Root Folder 2", userId = "2", parentFolderId = "1"))
+
+    `when`(folderRepository.getRootFoldersFromUid(eq("1"), any(), any())).then { invocation ->
+      val onSuccess = invocation.getArgument<(List<Folder>) -> Unit>(1)
+      onSuccess(userRootFoldersFlow)
+    }
+    folderViewModel.getRootFoldersFromUid("1")
+
+    composeTestRule.onNodeWithTag("showFileSystemButton").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("showFileSystemButton").performClick()
+    composeTestRule.onNodeWithTag("FileSystemPopup").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("FileSystemPopupFolderChoiceBox8").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("FileSystemPopupFolderChoiceBox9").assertIsDisplayed()
+    // composeTestRule.onNodeWithTag("FileSystemPopupFolderChoiceText").assertIsDisplayed()
```

Comment:
Maybe consider removing the commented lines if you are not using them

Reply:
Yes this was already removed in my latest push.